test/Tests/Strings/SequenceDistributionTests.cs

```diff
@@ -595,59 +595,41 @@ public void WordModel()
             var charDistUpperNarrow = ImmutableDiscreteChar.OneOf('A', 'B');
             var charDistLowerNarrow = ImmutableDiscreteChar.OneOf('a', 'b');
 
-            var charDistUpperScaled = charDistUpper.CreatePartialUniform(Math.Log(TargetProb1));
```

Comment:
Comment about use of non-normalized char distributions on the line 570 now needs an update.

Reply:
Fixed